design/design-doc-observe-only-resources.md

```diff
@@ -2,7 +2,7 @@
 
 * Owners: Hasan Turken (@turkenh)
 * Reviewers: Crossplane Maintainers
-* Status: Draft
+* Status: Defunct
```

Comment:
Is it worth adding a link to the ignore change one-pager to show that its been superseded by it?

Reply:
Good point